data/mods/MMA/professions.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "type": "profession",
+    "ident": "mma_battle_angel",
+    "name": "Battle Angel",
+    "description": "A combat cyborg built in the image of a girl...",
```

Comment:
You should probably avoid using gendered language here. This description would be odd for a male Battle Angel.

Reply:
Technically a born male human could still use a female-shaped robot body but you're probably right, best to avoid the subject unless actually desired.

Reply:
The body is capable of changing genders.  When we first see it in the maga, its male.  Also, angels can be male.